tests/test_default_qubit.py

```diff
@@ -297,12 +306,20 @@ class TestOperatorMatrices:
         ("Hadamard", np.array([[1, 1], [1, -1]])/np.sqrt(2)),
         ("CNOT", np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]])),
         ("SWAP", np.array([[1, 0, 0, 0], [0, 0, 1, 0], [0, 1, 0, 0], [0, 0, 0, 1]])),
+        ("CSWAP",np.array([[1, 0, 0, 0, 0, 0, 0, 0],
+                           [0, 1, 0, 0, 0, 0, 0, 0],
+                           [0, 0, 1, 0, 0, 0, 0, 0],
+                           [0, 0, 0, 1, 0, 0, 0, 0],
+                           [0, 0, 0, 0, 1, 0, 0, 0],
+                           [0, 0, 0, 0, 0, 0, 1, 0],
+                           [0, 0, 0, 0, 0, 1, 0, 0],
+                           [0, 0, 0, 0, 0, 0, 0, 1]])), 
```

Comment:
Todo: once #297 is done, we could feasibly do the following:

* Change the device API to instead recieve the entire Operation object, allowing access to attributes

* Use the provided matrix representations rather than having to redefine them here.

Question: this would significantly tidy up default.qubit. However, default.qubit now depends on PL core --- do we want this, or should it continue to be relatively 'independent'?

(This comment does not need to be taken into account for this PR!)